app/src/main/java/com/jerboa/ui/components/comment/reply/CommentReplyActivity.kt

```diff
@@ -68,76 +67,85 @@ fun CommentReplyActivity(
-                            )
-                        is ReplyItem.PostItem -> PostReply(
-                            postView = replyItem.item,
-                            account = account,
-                            reply = reply,
-                            onReplyChange = { reply = it },
-                            onPersonClick = { personId ->
-                                navController.navigate(route = "profile/$personId")
-                            },
-                            isModerator = isModerator(
-                                replyItem.item.creator,
-                                postViewModel
-                                    .moderators,
-                            ),
-                            modifier = Modifier
-                                .padding(padding)
-                                .imePadding(),
-                        )
-                        is ReplyItem.CommentReplyItem ->
-                            CommentReplyReply(
-                                commentReplyView = replyItem.item,
-                                account = account,
-                                reply = reply,
-                                onReplyChange = { reply = it },
-                                onPersonClick = { personId ->
-                                    navController.navigate(route = "profile/$personId")
-                                },
-                                modifier = Modifier
-                                    .padding(padding)
-                                    .imePadding(),
-                                showAvatar = siteViewModel.siteRes?.my_user?.local_user_view?.local_user?.show_avatars ?: true,
-                            )
-                        is ReplyItem.MentionReplyItem ->
-                            MentionReply(
-                                personMentionView = replyItem.item,
-                                account = account,
-                                reply = reply,
-                                onReplyChange = { reply = it },
-                                onPersonClick = { personId ->
-                                    navController.navigate(route = "profile/$personId")
-                                },
-                                modifier = Modifier
-                                    .padding(padding)
-                                    .imePadding(),
-                                showAvatar = siteViewModel.siteRes?.my_user?.local_user_view?.local_user?.show_avatars ?: true,
-                            )
+                when (val postRes = postViewModel.postRes) {
+                    is ApiState.Success -> {
+                        val moderators = postRes.data.moderators
+
+                        commentReplyViewModel.replyItem?.let { replyItem ->
+                            when (replyItem) {
+                                is ReplyItem.CommentItem ->
```

Comment:
`PersonProfileActivity` can reply to comments but does not use `postViewModel`. `InboxActivity` as well. So the `moderators` list might not be correct for the comment being edited? (From observation. Haven't verified.)

Reply:
I'll test this.

Reply:
Thx to @beatgammit for getting this one.